.travis.yml

```diff
@@ -198,18 +95,12 @@ addons:
     - sasl2-bin
     - swig
     - libnghttp2-dev
-    # documentation
-    - asciidoc
-    - asciidoctor
-    - dblatex
```

Comment:
We don't actually build docs in any of these jobs, and installing LaTeX takes long time.

Reply:
(this is a change that would also make sense in apache/qpid-dispatch).